package.json

```diff
@@ -19,24 +20,25 @@
   "license": "MIT",
   "homepage": "https://github.com/svelte/eslint-config#readme",
   "devDependencies": {
+    "@changesets/cli": "^2.27.1",
     "@svitejs/changesets-changelog-github-compact": "^1.1.0",
-    "@typescript-eslint/eslint-plugin": "^5.59.6",
-    "@typescript-eslint/parser": "^5.59.6",
-    "@changesets/cli": "^2.26.1",
-    "eslint": "^8.40.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-svelte": "^2.28.0",
-    "eslint-plugin-unicorn": "^47.0.0",
-    "typescript": "^5.0.4"
+    "eslint": "^9.0.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-svelte": "^2.36.0",
+    "eslint-plugin-unicorn": "^52.0.0",
+    "globals": "^15.0.0",
+    "svelte": "^4.2.12",
+    "typescript": "^5.4.4",
+    "typescript-eslint": "^7.6.0"
   },
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": ">= 5",
-    "@typescript-eslint/parser": ">= 5",
-    "eslint": ">= 8",
-    "eslint-config-prettier": ">= 8",
-    "eslint-plugin-svelte": ">= 2",
-    "eslint-plugin-unicorn": ">= 47",
-    "typescript": ">= 4"
+    "eslint": ">= 9",
+    "eslint-config-prettier": ">= 9",
+    "eslint-plugin-svelte": ">= 2.36",
```

Comment:
is there a reason we don't leave this as `>= 2`?

Reply:
https://github.com/sveltejs/eslint-plugin-svelte/releases/tag/v2.36.0 added support for ESLint 9 / flat config.